CHANGES.md

```diff
@@ -1,7 +1,7 @@
 # Changes
 
 ### v0.5.0
-
+* Add `plot_disparities_in_performance` and `plot_disparities_in_selection_rate` functions https://github.com/fairlearn/fairlearn/pull/561
```

Comment:
nit: Add 'plots' module with ...

Reply:
Oh just saw you renamed it ti metrics_plots. I liked the original name better but obviously that's very subjective